### DISCUSSION TITLE
微调出现的问题

### DISCUSSION THREAD

**Discussion by yodascraft** [zai-org/ChatGLM3#472] [General]
![image](https://github.com/THUDM/ChatGLM3/assets/84711268/2e01673c-18c8-4deb-a288-e6092a66f5bb)
出现了keyError:使用环境是python3.9.
数据集是推荐的ToolAlpaca 数据集

**Comment by zRzRzRzRzRzRzR** [comment]
请在 #253  讨论，其次，python升级到3.10
